.gitignore

```
@@ -382,3 +382,6 @@ FodyWeavers.xsd
 /Setup/setup.7z
 Assembly/main.bin
 /add to release
+exported_symbols.txt
+patchData.json
+enemizerBasePatch.json
```

These entries will only ignore the files in the root directory of the repo. If you want to include all subdirectories check out the suggestion. I am unsure if this is intentional, so I figured I'd say something.

```suggestion
**/exported_symbols.txt
**/patchData.json
**/enemizerBasePatch.json
```

---

Commited and resolved